linux/scripts/deb-packaging.sh

```diff
@@ -50,25 +48,23 @@ if builder_start_action source; then
   echo "${END_STEP}"
 
   mv builddebs/* "${OUTPUT_PATH:-..}"
+}
 
-  builder_finish_action success source
-  exit 0
-fi
-
-if builder_start_action verify; then
+verify_action() {
   tar xf "${SRC_PKG}"
-  if [ ! -f debian/libkeymancore.symbols ]; then
-    echo ":warning: Missing libkeymancore.symbols file" >&2
+  PKG_NAME=libkeymancore
+  LIB_NAME=libkeymancore
+  if [ ! -f debian/${PKG_NAME}.symbols ]; then
+    echo ":warning: Missing ${PKG_NAME}.symbols file" >&2
   else
-    PKG_NAME=libkeymancore
-    LIB_NAME=libkeymancore
-
     tmpDir=$(mktemp -d)
     dpkg -x "${BIN_PKG}" "$tmpDir"
     cd debian
     dpkg-gensymbols -v"${PKG_VERSION}" -p${PKG_NAME} -e"${tmpDir}"/usr/lib/x86_64-linux-gnu/${LIB_NAME}.so* -O${PKG_NAME}.symbols -c4
     echo ":heavy_check_mark: ${LIB_NAME} API didn't change" >&2
   fi
-  builder_finish_action success verify
-  exit 0
-fi
+}
+
+builder_run_action dependencies  dependencies_action
+builder_run_action source        source_action
+builder_run_action verify        verify_action
```

Comment:
I notice you are no longer doing `exit 0` after each action. Is this change in behaviour intentional?

Reply:
Yes, this brings it more in line with how the other `build.sh` scripts work. I can't see a reason why you shouldn't be able to run multiple actions.

Reply:
Sounds sensible to me. Just wasn't sure if there was a particular reason it was doing that before :grin: